hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -654,4 +662,34 @@ private void writeChunkToContainer(ChunkBuffer chunk) throws IOException {
   public void setXceiverClient(XceiverClientSpi xceiverClient) {
     this.xceiverClient = xceiverClient;
   }
+
+  /**
+   * Handles InterruptedExecution.
+   *
+   * @param ex
+   * @param skipExecutionException is optional, if passed as TRUE, then
+   * handle ExecutionException else skip it.
+   * @throws IOException
+   */
+  private void handleInterruptedException(Exception ex,
+      boolean skipExecutionException)
+      throws IOException {
+    LOG.error("Command execution was interrupted.");
+    if(skipExecutionException) {
```

Comment:
Minor NIT: variable name is called skipExecutionException, but when true, we are handling ExecutionException. I think it should be viceversa.

Reply:
Addressed in latest commit.